cypress/integration/UserPersonalDetails.feature

```diff
@@ -1,6 +1,15 @@
 Feature: Change User Personal Details
 
     As a user of Make an Lpa I want to be able to change my personal details
+
+    # NOTE: this test is not idempotent - you will get a failure the second time you run it
+    # TODO reset address in test data before running the test
+
```

Comment:
Why does it fail 2cnd time and is there a way to avoid that happening?

Reply:
Because the test changes the address and doesn't change it back again at the end. It would be better if we could poke the necessary data into the test before running it, but the lack of API access prevents that from happening.

Reply:
Could the test be made to change it back at the end?  So as to leave things as they were?  Or is that too much of a pain?  (haven't looked at the code)

Reply:
It would be possible, but not in scope for this ticket. I put the note in there to highlight to anyone checking this manually. However, could you please raise a ticket to improve this test as you explained?

Reply:
OK fair enough :)